cypress/e2e/users_spec/UsersCreation.cy.ts

```diff
@@ -126,20 +126,21 @@ describe("User Creation", () => {
   });
 
   it("Update the existing user profile Form Mandatory File Error", () => {
-    userCreationPage.clickElementById("user-profile-name");
-    userCreationPage.clickElementById("profile-button");
-    userCreationPage.clickElementById("edit-cancel-profile-button");
+    userCreationPage.clickProfileName();
+    userCreationPage.clickProfileButton();
+    userCreationPage.clickEditCancelProfileButton();
+
     userCreationPage.clearIntoElementById("firstName");
     userCreationPage.clearIntoElementById("lastName");
     userCreationPage.clearIntoElementById("phoneNumber");
     userCreationPage.clearIntoElementById("altPhoneNumber");
     userCreationPage.clearIntoElementById("weekly_working_hours");
-    userCreationPage.clickElementById("submit");
+    cy.submitButton("Update");
     userCreationPage.verifyErrorMessages(EXPECTED_PROFILE_ERROR_MESSAGES);
   });
 
   it("create new user and verify reflection", () => {
-    userCreationPage.clickElementById("addUserButton");
+    userCreationPage.clickAddUserButton();
     userCreationPage.selectFacility("Dummy Shifting Center");
     userCreationPage.typeIntoElementById("username", username);
     userCreationPage.typeIntoElementById("password", "Test@123");
```

Comment:
remove all usage of typeintoelementbyId and create POM approach based individual functions

Reply:
Done